srml/contracts/rpc/src/lib.rs

```diff
@@ -121,4 +138,57 @@ where
 
 		Ok(exec_result)
 	}
+
+	fn get_storage(
+		&self,
+		address: AccountId,
+		key: Bytes,
```

Comment:
Why not use `H256` if you need to check the length anyway?

Reply:
Oh, didn't realize that it is an option!